purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/trustedentitlements/TrustedEntitlementsInformationalModeIntegrationTest.kt

```diff
@@ -35,20 +36,12 @@ class TrustedEntitlementsInformationalModeIntegrationTest : BasePurchasesIntegra
     }
 
     @Test
-    fun initialCustomerInfoIsVerified() {
-        var receivedCustomerInfo: CustomerInfo? = null
-        ensureBlockFinishes { latch ->
-            Purchases.sharedInstance.getCustomerInfoWith(
-                onError = { fail("should be success. Error: ${it.message}") },
-                onSuccess = {
-                    receivedCustomerInfo = it
-                    latch.countDown()
-                },
-            )
-        }
+    fun initialCustomerInfoIsVerifiedAwaitCustomerInfo() {
+        activity.lifecycleScope.launch {
```

Comment:
Is there a way to avoid repeating this in every test?

Reply:
Good question. Let me think about it and will report back here.

Reply:
Abstracted it a little by extracting it into 👇

https://github.com/RevenueCat/purchases-android/blob/8d52048c3e34efc951fd2bc862f486af7028389b/purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/BasePurchasesIntegrationTest.kt#L172-L178

Please let me know what you think.